### DISCUSSION TITLE
[Projects Beta] Separation of concerns Table vs Board

### DISCUSSION THREAD

**Discussion by droopy12** [community/community#6688] [Projects and Issues] [answered]
I just tried out the Projects Beta version and as someone who is setting up project management with issues, I highly appreciate it. 

Nevertheless, I see the Boards view no longer has the detailed information it used to (labels, milestones, side preview) based on the assumption(?) that the users will find all the information they need in the Table view. 

And even though for my project management tasks the table view is really exteremely useful, on a day-to-day basis our team would still prefer to walk through a Kanban style board and keep it as simple as possible. 

In a nutshell, for me the two views have different audiences with their own needs and goals and the current implementation seems more focused on the project management part rather than the daily development one.

**Comment by timoyates** [comment, marked as answer]
@droopy12 labels, milestones and various other fields can be enabled for board cards if you click into the view's setting menu and toggle them on 😊

**Reply by droopy12** [reply]
Thanks @timoyates! I hadn't noticed but I will check it out.

**Reply by timoyates** [reply]
Great! The need to preview issues or see more of their content without opening a new tab is on our roadmap too. Mind if I mark this as answered? 😇

**Reply by droopy12** [reply]
Just tried it out and it looks great. Thanks!
